src/app/app-routing.module.ts

```diff
@@ -0,0 +1,40 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+
+const routes: Routes = [
+  {
+    path: 'login',
+    loadChildren: './login/login.module#LoginModule'
+  },
+  {
+    path: 'shell',
+    loadChildren: './shell/shell.module#ShellModule'
+  },
+  {
+    path: 'admin-panel',
```

Comment:
app router should have only login and shell router

other routers need to be moved to shell router

Reply:
Done!